composer.json

```diff
@@ -54,7 +54,8 @@
         "netgen/layouts-ibexa-site-api": "~1.4.0",
         "netgen/layouts-ibexa-relation-list-query": "^1.4",
         "netgen/layouts-ibexa-tags-query": "^1.4",
-        "netgen/conversations": "^1.0"
+        "netgen/conversations": "^1.0",
+        "netgen/notifications": "*"
```

Comment:
We should limit this to a version, probably same as `netgen/conversations` above.

Reply:
Done - https://github.com/netgen/media-site/pull/189/commits/cdac9340011ec2bd393e2fe00b1002b235e96bf0